assets/js/moneybadger-checkout.js

```diff
@@ -0,0 +1,18 @@
+jQuery(document).ready(function($) {
+    setInterval(function() {
+        $.ajax({
+            url: moneybadger_checkout_params.order_status_url,
+            type: 'get',
+            data: {},
+            success: function(response) {
+                console.log(response);
+                if (
+                  response.hasOwnProperty('order_is_complete') &&
+                  response.order_is_complete
+                ) {
+                    window.location.href = moneybadger_checkout_params.order_complete_url;
+                }
+            },
+        });
+    }, 3000);
```

Comment:
this is a very slow poll. our API supports long polling via the `wait` parameter, so using that will result in a much better user experience

Reply:
the MB payment page also allows you to specify a "return" parameter which will automatically redirect after payment is complete, no sure if that helps?

Reply:
@ckritzinger  i went with the 500ms poll. unless you'll set security on your domain to allow external sites to detect navigation, or perhaps to add a postMessage for when the payment state changes?